go.mod

```diff
@@ -28,11 +25,11 @@ require (
 	github.com/uber/jaeger-lib v1.5.0 // indirect
 	go.uber.org/atomic v1.3.2 // indirect
 	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
-	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
-	k8s.io/api v0.0.0-20181128191700-6db15a15d2d3
-	k8s.io/apiextensions-apiserver v0.0.0-20190118124337-a384d17938fe
-	k8s.io/apimachinery v0.0.0-20181128191346-49ce2735e507
-	k8s.io/client-go v2.0.0-alpha.0.0.20190115164855-701b91367003+incompatible
+	k8s.io/api v0.0.0-20190222213804-5cb15d344471
+	k8s.io/apiextensions-apiserver v0.0.0-20190508191920-007dc40467c5
+	k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
+	k8s.io/client-go v2.0.0-alpha.0.0.20190507014756-65905f29c17c+incompatible
```

Comment:
I noticed when upgrading my own controller to Kooper's master branch, that it still pulls client-go v2.0.0.

Is there a reason why you're not using `client-go@v1.10.0`? 
Asking out of curiosity - it seems to work fine 🙂

Reply:
TL;DR: it's using `kubernetes-1-13-6` version ([this]).

That's the strange go modules notation, not the library tag itself, the way Kooper sets the libraries version is using this:

https://github.com/spotahome/kooper/blob/bb50a407ffa98e2926d1e632b698a00e04708f87/Makefile#L30-L37

So I set `kubernetes-1-13-6` version ([this]) on go mod and when `go mod tidy`-ing, it changes to its strange notation, check the gomod `65905f29c17c` signature and check the commit of that tag that points to client-go version ([this]).

(also the 2.0.0 notation on client-go is from 2016, would not compile anything on Kooper)

[this]: https://github.com/kubernetes/client-go/tree/kubernetes-1.13.6

Reply:
Oh okay, interesting (and kinda weird 😄)
Thanks for the explanation - I should've had a closer look at the client-go signature (`v2.0.0-alpha.0.0.20190507014756-65905f29c17c+incompatible`), which is obviously a commit from 2019 🤦‍♂

Reply:
Yep, is weird :laughing: